CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.3.0] - 2022-1-18
 ## Changed
 - Always save data to data group on write in StorageDataGroup
+- Prevent persist from exiting early, when task is kicked off
```

Comment:
1.3.0 and 1.3.1 have already released, so let's put this under 1.3.2

Reply:
Bumped!

Reply:
The release process is:
* Update CHANGELOG with release date
* Run `npm version` (in this case `npm version patch`)
* Push `master` and new tag to GitHub
* Run `npm publish --access=public`